src/SSVOpenHexagon/Core/HexagonGame.cpp

```diff
@@ -190,16 +190,18 @@ void HexagonGame::incrementDifficulty()
 {
     assets.playSound("levelUp.ogg");
 
+    const int signMult = ssvu::getSign(levelStatus.rotationSpeed) == 0 ? 1 : ssvu::getSign(levelStatus.rotationSpeed);
```

Comment:
```suggestion
    const float signMult = (levelStatus.rotationSpeed > 0.f) ? 1.f : -1.f;
```

Wouldn't this work and be simpler?

Reply:
Fair enough. And it lowers reliance on SSVU.